protoc-gen-swagger/genswagger/template.go

```diff
@@ -662,15 +659,11 @@ func renderServices(services []*descriptor.Service, paths swaggerPathsObject, re
 					} else {
 						lastField := b.Body.FieldPath[len(b.Body.FieldPath)-1]
 						schema = schemaOfField(lastField.Target, reg, customRefs)
-						if schema.Description != "" {
-							desc = schema.Description
-						} else {
-							desc = fieldProtoComments(reg, lastField.Target.Message, lastField.Target)
-						}
 					}
 
+					desc := ""
 					if meth.GetClientStreaming() {
-						desc += " (streaming inputs)"
```

Comment:
This should have the space. This should have the +.

Come to think of it, this looks like it MIGHT be a rollback of a pull request I approved recently. Why wouldn't you want to have `fieldProtoComments` result used for desc? (This might be the reason for disappearing comments in the autogenerated code above.)

Same applies for many other changes in this file: they look like rollbacks. Please don't rollback needlessly and please rebase on master correctly without steamrolling older changes :-)